src/components/IconButton/IconButton.tsx

```diff
@@ -7,7 +7,7 @@ import Icon from '../Icon';
 import { AcceptedIconNames } from '../Icon/types';
 import ButtonBase, { ButtonBaseProps } from 'components/ButtonBase/ButtonBase';
 
-export type IconButtonProps = Omit<ButtonBaseProps, 'isIconButton' | 'iconLeft' | 'iconRight'> & {
+export type IconButtonProps = Omit<ButtonBaseProps, 'iconLeftName' | 'iconRightName'> & {
```

Comment:
Icon doesnt work as expected 
![Screenshot 2023-01-27 at 2 27 46 PM](https://user-images.githubusercontent.com/6433679/215086973-787619f8-5978-4c3e-a923-b451bda5aad2.png)

Reply:
IconButton will be implemented in a dedicated future PR